pkg/apis/crd/v1alpha1/types.go

```diff
@@ -591,3 +596,17 @@ type TierList struct {
 
 	Items []Tier `json:"items"`
 }
+
+type ServiceAccount struct {
```

Comment:
Maybe we should name it PeerServiceAccount or PeerServiceAccounts if it can select multiple ServiceAccounts to be consistent with PeerNamespaces.

Reply:
Changed.